message.xml

```diff
@@ -4,7 +4,7 @@ this repository contains the full copyright notices and license terms. -->
 <tryton>
     <data grouped="1">
         <record model="ir.message" id="msg_can_not_revoke">
-            <field name="text">You cannot revoke "%(record)s" because must first cancel, draft or done the moves or shipments:
+            <field name="text">You cannot revoke "%(record)s" because you there are moves that are not in cancel, draft or done state:
```

Comment:
The second "you" should not be there.

Also usually the name of the states shoul be capitalized:

```
You cannot revoke "%(record)s" because there are moves that are not in Cancel, Draft or Done state:"
```

Reply:
aa3e39a50ccce7e3a618577f957251e12edd6bf3